style.css

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
it is good practice when you create reset default styles. For example, * {....}

Reply:
Fixed (have added reset.css)